README.md

```diff
@@ -14,75 +20,66 @@
   <br/>  
-  <br/>  
-  <br/>  
-  <br/>  
-
-  <p align="center">
-    <img src="design/feature-panel.png"  title="Immich Logo">
-  </p>
+  <br/>   
 </p>
 
-# Immich
+## Content
+- [Features](#features)
+- [Screenshots](#screenshots)
+- [Installation](#installation)
+- [Mobile App](#-mobile-app)
+- [Developers](#development)
+- [Support](#support)
+- [Known Issues](#known-issues)
 
-**High performance self-hosted photo and video backup solution.**
-
-![](https://media.giphy.com/media/y8ZeaAigGmNvlSoKhU/giphy.gif)
+# Features 
 
-Loading ~4000 images/videos
+> ⚠️ WARNING: **NOT READY FOR PRODUCTION! DO NOT USE TO STORE YOUR ASSETS**. This project is under heavy development, there will be continuous functions, features and api changes.
 
-## Screenshots
+|  | Mobile | Web |
+| - | - | - | 
+| ☁️ Upload and view videos and photos | Yes | Yes 
+| 🔄 Auto backup when the app is opened | Yes | N/A
+| ☑️ Selective album(s) for backup | Yes | N/A
+| ⬇️ Download photos and videos to local device | Yes | Yes
+| 👪 Multi-user support | Yes | Yes
+| 🖼️ Album | Yes | Yes
+| 🤝 Shared Albums | Yes | Yes
+| 🚀 Quick navigation with draggable scrollbar | Yes | Yes
+| 🗃️ Support RAW (HEIC, HEIF, DNG, Apple ProRaw) | Yes | Yes
+| 🧭 Metadata view (EXIF, map) | Yes | Yes
+| 🔎 Search by metadata, objects and image tags | Yes | No
+| ⚙️ Administrative functions (user management) | N/A | Yes
 
-### Mobile
-<p align="left">
-  <img src="design/login-screen.png" width="150" title="Login With Custom URL">
-  <img src="design/backup-screen.png" width="150" title="Backup Setting Info">
-  <img src="design/selective-backup-screen.png" width="150" title="Backup Setting Info">
-  <img src="design/home-screen.jpeg" width="150" title="Home Screen">
-  <img src="design/search-screen.jpeg" width="150" title="Curated Search Info">
-  <img src="design/shared-albums.png" width="150" title="Shared Albums">
-  <img src="design/nsc6.png" width="150" title="EXIF Info">
-</p>
 
-### Web
-<p align="left">
-  <img src="design/web-home.jpeg"  width="49%" title="Home Dashboard">
-  <img src="design/web-detail.jpeg" width="49%" title="Detail">
-</p>
-
-# Note
+  <br/>  
 
-**!! NOT READY FOR PRODUCTION! DO NOT USE TO STORE YOUR ASSETS !!**
+# Screenshots
 
-This project is under heavy development, there will be continuous functions, features and api changes.
+### Mobile
+| | | | | |
+| - | - | - | - | - |
+| <img src="design/login-screen.png" width="150" title="Login With Custom URL"> <p align="center"> Login with custom URL </p> | <img src="design/backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup Settings </p> | <img src="design/selective-backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup tunning </p> | <img src="design/home-screen.jpeg" width="150" title="Home Screen"> <p align="center"> Home Screen </p> | <img src="design/search-screen.jpeg" width="150" title="Curated Search Info"> <p align="center"> Curated search </p> |
```

Comment:
Typo: "Backup tunning" should be "Backup tuning" however I think maybe "Backup Selection" is better?